C/tests/c4Test.cc

```diff
@@ -627,23 +627,16 @@ void C4Test::checkAttachments(C4Database *inDB, vector<C4BlobKey> blobKeys, vect
 
 #pragma mark - FILE IMPORT:
 
-
-// Reads a file into memory.
-fleece::alloc_slice C4Test::readFile(std::string path) {
-    INFO("Opening file " << path);
-    FILE *fd = fopen(path.c_str(), "rb");
-    REQUIRE(fd != nullptr);
-    fseeko(fd, 0, SEEK_END);
-    auto size = (size_t)ftello(fd);
-    fseeko(fd, 0, SEEK_SET);
-    fleece::alloc_slice result(size);
-    ssize_t bytesRead = fread((void*)result.buf, 1, size, fd);
-    REQUIRE(bytesRead == size);
-    fclose(fd);
+// Parameter is relative filepath for cert from project root
+fleece::alloc_slice C4Test::readFile(std::string filepath) {
+    std::ifstream inFile(filepath);
+    REQUIRE(inFile.is_open());
+    std::stringstream outData;
+    outData << inFile.rdbuf();
```

Comment:
Check io read by size, as something like, " REQUIRE(bytesRead == size); " in the above.